src/libs/actions/user.ts

```diff
@@ -42,6 +42,54 @@ export async function saveUser(user: User) {
   }
 }
 
+export async function creditUser(clientId: string) {
```

Comment:
this is used only for initial user credit, either 
1. Take credit to add from the params, or
2. rename this method specific to initUserCredits
i prefer option 1